website/public/js/app.js

```diff
@@ -150,12 +150,17 @@ window.habitrpg = angular.module('habitrpg',
           url: '/:gid',
           templateUrl: 'partials/options.social.guilds.detail.html',
           title: env.t('titleGuilds'),
-          controller: ['$scope', 'Groups', 'Chat', '$stateParams',
-          function($scope, Groups, Chat, $stateParams){
-            Groups.Group.get({gid:$stateParams.gid}, function(group){
-              $scope.group = group;
-              Chat.seenMessage(group._id);
-            });
+          controller: ['$scope', 'Groups', 'Chat', '$stateParams', 'Members',
+          function($scope, Groups, Chat, $stateParams, Members){
+            Groups.Group.get($stateParams.gid)
+              .then(function (response) {
+                $scope.group = response.data.data;
+                Chat.seenMessage($scope.group._id);
+                Members.getGroupMembers($scope.group._id)
```

Comment:
Where do we display invites on the UI? In the member section? If so, we should definitely fetch them here.